src/MenuBar.jsx

```diff
@@ -65,16 +65,25 @@ export class homeMenuBar extends React.Component {
       <Navbar collapseOnSelect className={styles.navigationbar}>
         <Navbar.Header>
           <Navbar.Brand className={styles.navbarheader}>
-            <a href="#">Midtown Assistance Center</a>
+            <a href="/dash">Midtown Assistance Center</a>
           </Navbar.Brand>
           <Navbar.Toggle />
         </Navbar.Header>
         <Navbar.Collapse>
           <Nav pullLeft>
-            <NavItem className={styles.navbartext}>About us</NavItem>
+            {window.location.pathname === '/dash/about' ?
+            <MenuItem className={styles.navbartext} href="/dash">Dashboard</MenuItem>
+            :
+            <MenuItem className={styles.navbartext} href="/dash/about">About us</MenuItem>}
           </Nav>
           <Nav pullRight>
-            <MenuItem onClick={() => this.logout()} className={styles.navbartext}>{this.props.user}</MenuItem>
+            <span><img className="nav-prof-pic" src='/images/user.png'></img></span>
```

Comment:
We should decide how we'll be storing the image, as this affects rendering. Will we store links to files on S3? Or compress the images in base64 strings and just store them on MongoDB (works well for low-res pictures)? Probably a later problem...

Reply:
We should probably ask MAC about this also b/c there's a cost associated with S3. I think we should start with just using Mongo.